magefiles/main.go

```diff
@@ -147,6 +150,9 @@ func LocalDev(arg string) error {
 	case "minimal":
 		os.Setenv("ARMADA_COMPONENTS", "executor,server")
 		mg.Deps(StartComponents)
+	case "minimal-pulsar":
+		os.Setenv("ARMADA_COMPONENTS", "executor-pulsar,server-pulsar,postgresPulsarMigration,scheduler,scheduleringester")
```

Comment:
Why isn't `eventingester` in this list? Surely you'd need it started to get events

Reply:
Because the scheduler depends on the event ingester. Technically you don't need anything in that list if it's depended on in the docker compose file.